assets/src/edit-story/components/errorBoundary/index.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ErrorActions from './errorActions';
+
+class ErrorBoundary extends Component {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+  };
+
+  state = {
+    hasError: false,
+  };
+
+  static getDerivedStateFromError() {
+    return { hasError: true };
+  }
+
+  // eslint-disable-next-line no-unused-vars
+  componentDidCatch(error, errorInfo) {
+    // In the future we will take some action here
```

Comment:
Can you pass `error` and `errorInfo` to `ErrorActions`?

Reply:
Something like this? (it overrides the template, AFAIK we can't populate "placeholders" in the template, but we can insert the template on our side).

![image](https://user-images.githubusercontent.com/21036224/85126257-6a01d980-b22d-11ea-9ad9-29e6d02bc193.png)

Reply:
Excellent!

Let's not worry about the template for now. We'll probably use a different URL (e.g. a Google form) anyway, but it's important that we can pass the error to that. So, thanks!